apps/financial-aid/web-veita/src/components/ApplicationsTable/ApplicationsTable.tsx

```diff
@@ -8,50 +8,88 @@ import cn from 'classnames'
 import {
   TableHeaders,
   TableBody,
+  LoadingContainer,
+  TableSkeleton,
 } from '@island.is/financial-aid-web/veita/src/components'
-import { Application } from '@island.is/financial-aid/shared/lib'
+import {
+  Application,
+  ApplicationState,
+  ApplicationStateUrl,
+} from '@island.is/financial-aid/shared/lib'
 import { TableHeadersProps } from '@island.is/financial-aid-web/veita/src/routes/ApplicationsOverview/applicationsOverview'
 
+import { useAllApplications } from '@island.is/financial-aid-web/veita/src/utils/useAllApplications'
+
 interface PageProps {
   applications: Application[]
+  setApplications: React.Dispatch<
+    React.SetStateAction<Application[] | undefined>
```

Comment:
Can we change this type to not have `undefined`?

Reply:
But application can be undefined, if we cant get to the backend or some? 🤔